Aztec/Classes/TextKit/TextView.swift

```diff
@@ -887,9 +887,13 @@ open class TextView: UITextView {
         let delay = 0.05
         DispatchQueue.main.asyncAfter(deadline: .now() + delay) {
             let pristine = self.selectedRange
-            let updated = NSMakeRange(max(pristine.location - 1, 0), 0)
+            var location = max(pristine.location - 1, 0)
+            if pristine.location == location {
+                location = min(pristine.location + 1, self.storage.length)
```

Comment:
Ahhhhh i see! Goal there is to move the **selectedRange.location** to +1 or -1, in order to force the cursor to redraw. Logic is as follows:

If we're at the end of the document... we can't move +1, or we'd trigger an Out of Bounds exception, in which case we'll proceed with -1.

(After seeing this logic a couple days later, i think it can be just simplified). Will be updating + documenting why, thanks Diego!